packages/consent/consent-wrapper-trustarc/README.md

```diff
@@ -0,0 +1,3 @@
+# @segment/analytics-consent-wrapper-trustarc
+
+## TODO
```

Comment:
Still need to create a README file with documentation on how to use TrustArc's wrapper then deploying TrustArc's script.